sympy/parsing/sympy_parser.py

```diff
@@ -507,6 +509,95 @@ def implicit_multiplication_application(result, local_dict, global_dict):
 
     return result
 
+def find_closing(string,pos,opening,closing):
```

Comment:
Can you use the `_group_parentheses` function here instead (after a modification to make the brace type a keyword argument)?